resources/qml/ExtruderIcon.qml

```diff
@@ -17,29 +17,26 @@ Item
     property alias textColor: extruderNumberText.color
     property bool extruderEnabled: true
 
-    UM.RecolorImage
+    Item
     {
-        id: mainIcon
+        opacity: extruderEnabled ? 1 : 0.4
         anchors.fill: parent
 
-        source: UM.Theme.getIcon("Extruder", "medium")
-        color: extruderEnabled ? materialColor: UM.Theme.getColor("disabled")
-    }
-
-    Rectangle
-    {
-        id: extruderNumberCircle
-
-        width: height
-        height: Math.round(parent.height / 2)
-        radius: Math.round(width)
-        color: UM.Theme.getColor("toolbar_background")
+        UM.RecolorImage
+        {
+            id: blorp
```

Comment:
This is not a very... descriptive ID. It's not used anywhere either. Better remove it.

Reply:
WHoops. My bad. I copied it and it was complaining about a duplicate.

Reply:
blorp, forever in our hearts